### PR TITLE
gtfs_reader - add tests on agencies and fix agency_id default

### PR DESCRIPTION
**Pull request by prhod**

### CODE REVIEW

File: src/gtfs.rs [hunk 1]
```diff
@@ -20,8 +20,13 @@ struct Agency {
 }
 impl From<Agency> for objects::Network {
     fn from(agency: Agency) -> objects::Network {
+        let agency_id: String;
+        match agency.id {
+            None => agency_id = default_agency_id(),
+            Some(id) => agency_id = id,
+        }
```
Comment (TeXitoi) [inline src/gtfs.rs]:
```rust
let agency_id = match agency.id {
    None => default_agency_id(),
    Some(id) => id,
};
```

File: tests/gtfs_reader.rs [hunk 2]
```diff
@@ -18,6 +19,8 @@ fn load_minimal_agency() {
     let networks = gtfs::read_agency(tmp_dir.path());
     tmp_dir.close().expect("delete temp dir");
     assert_eq!(1, networks.len());
+    let agency = &networks.into_vec()[0];
```
Comment (TeXitoi) [inline tests/gtfs_reader.rs]:
```rust
let agency = networks.iter().next().unwrap().1;
```
Minor for this case, but if you want to do something else with networks, `into_vec` will be problematic.

File: tests/gtfs_reader.rs [hunk 3]
```diff
@@ -46,4 +49,20 @@ id_1,My agency,http://my-agency_url.com,Europe/London,EN,0123456789,http://my-ag
...
+fn load_2_agencies_with_no_id() {
+    let agency_content = "agency_name,agency_url,agency_timezone\n
+My agency 1,http://my-agency_url.com,Europe/London
+My agency 2,http://my-agency_url.com,Europe/London";
+    let tmp_dir = TempDir::new("navitia_model_tests").expect("create temp dir");
+    let file_path = tmp_dir.path().join("agency.txt");
+    let mut f = File::create(&file_path).unwrap();
+    f.write_all(agency_content.as_bytes()).unwrap();
+    gtfs::read_agency(tmp_dir.path());
+    tmp_dir.close().expect("delete temp dir");
```
Comment (TeXitoi) [inline tests/gtfs_reader.rs]: 👍

File: src/gtfs.rs [hunk 4]
```diff
@@ -20,8 +20,13 @@ struct Agency {
...
         objects::Network {
-            id: agency.id,
+            id: agency_id,
```
Comment (datanel) [inline src/gtfs.rs]: I think this could work
```rust
id: agency.id.unwrap_or(default_agency_id())
```

Comment (TeXitoi) [reply src/gtfs.rs]:
```rust
id: agency.id.unwrap_or_else(default_agency_id),
```

File: tests/gtfs_reader.rs [hunk 5]
```diff
@@ -1,4 +1,5 @@
 extern crate navitia_model;
+extern crate serde_json;
```
Comment (TeXitoi) [inline tests/gtfs_reader.rs]: I don't think that's needed.

**Pull request merged**